docs/install.rst

```diff
@@ -245,8 +245,8 @@ packages. They are part of `GTK+ <https://en.wikipedia.org/wiki/GTK+>`_
 The following installation instructions for the GTK+ libraries don't work on
 Windows XP. That means: Windows Vista or later is required.
 
-Of course you can decide to install ancient WeasyPrint versions with an
-erstwhile Python, combine it with outdated GTK+ libraries on any Windows
+Of course you can decide to install older WeasyPrint versions with an
+older Python versions, combine it with outdated GTK+ libraries on any Windows
```

Comment:
I liked `erstwhile`and `ancient`. In fact: I explicitly tried to avoid boring `older`.